ev3dev2/sensor/lego.py

```diff
@@ -727,48 +727,48 @@ class InfraredSensor(Sensor, ButtonBase):
     on_channel2_bottom_left = None
-    #: Handler for top-right button events on channel 2. See :meth:`InfraredSensor.process`.
+    #: Handler for top-right button events on channel 2. See :meth:``InfraredSensor.process``.
     on_channel2_top_right = None
-    #: Handler for bottom-right button events on channel 2. See :meth:`InfraredSensor.process`.
+    #: Handler for bottom-right button events on channel 2. See :meth:``InfraredSensor.process``.
     on_channel2_bottom_right = None
-    #: Handler for beacon button events on channel 2. See :meth:`InfraredSensor.process`.
+    #: Handler for beacon button events on channel 2. See :meth:``InfraredSensor.process``.
     on_channel2_beacon = None
 
-    #: Handler for top-left button events on channel 3. See :meth:`InfraredSensor.process`.
+    #: Handler for top-left button events on channel 3. See :meth:``InfraredSensor.process``.
     on_channel3_top_left = None
-    #: Handler for bottom-left button events on channel 3. See :meth:`InfraredSensor.process`.
+    #: Handler for bottom-left button events on channel 3. See :meth:``InfraredSensor.process``.
     on_channel3_bottom_left = None
-    #: Handler for top-right button events on channel 3. See :meth:`InfraredSensor.process`.
+    #: Handler for top-right button events on channel 3. See :meth:``InfraredSensor.process``.
     on_channel3_top_right = None
-    #: Handler for bottom-right button events on channel 3. See :meth:`InfraredSensor.process`.
+    #: Handler for bottom-right button events on channel 3. See :meth:``InfraredSensor.process``.
     on_channel3_bottom_right = None
-    #: Handler for beacon button events on channel 3. See :meth:`InfraredSensor.process`.
+    #: Handler for beacon button events on channel 3. See :meth:``InfraredSensor.process``.
     on_channel3_beacon = None
 
-    #: Handler for top-left button events on channel 4. See :meth:`InfraredSensor.process`.
+    #: Handler for top-left button events on channel 4. See :meth:``InfraredSensor.process``.
     on_channel4_top_left = None
-    #: Handler for bottom-left button events on channel 4. See :meth:`InfraredSensor.process`.
+    #: Handler for bottom-left button events on channel 4. See :meth:``InfraredSensor.process``.
     on_channel4_bottom_left = None
-    #: Handler for top-right button events on channel 4. See :meth:`InfraredSensor.process`.
+    #: Handler for top-right button events on channel 4. See :meth:``InfraredSensor.process``.
     on_channel4_top_right = None
-    #: Handler for bottom-right button events on channel 4. See :meth:`InfraredSensor.process`.
+    #: Handler for bottom-right button events on channel 4. See :meth:``InfraredSensor.process``.
     on_channel4_bottom_right = None
-    #: Handler for beacon button events on channel 4. See :meth:`InfraredSensor.process`.
+    #: Handler for beacon button events on channel 4. See :meth:``InfraredSensor.process``.
```

Comment:
Same as other comment (might be additional instances); I think it might be required as-is

Reply:
I’m not sure on these two will revert these parts to play it safe and then merge.